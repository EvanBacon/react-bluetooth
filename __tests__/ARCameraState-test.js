import React from 'react';
import renderer from 'react-test-renderer';

import ARCameraStateIOS from '../lib/ARCameraState.ios.js';
import ARCameraStateDefault from '../lib/ARCameraState.js';

describe('ARCameraState', () => {
  describe('Platform.default', () => {
    it('renders', () => {
      const tree = renderer.create(<ARCameraStateDefault />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
  describe('Platform.ios', () => {
    it('renders', () => {
      const tree = renderer.create(<ARCameraStateIOS />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
