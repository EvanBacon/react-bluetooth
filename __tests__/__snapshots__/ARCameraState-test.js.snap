// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ARCameraState Platform.default renders 1`] = `null`;

exports[`ARCameraState Platform.ios renders 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "left": 12,
        "position": "absolute",
        "right": 12,
        "top": 56,
      },
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "fontSize": 24,
          "fontWeight": "bold",
          "textAlign": "center",
        },
        Object {
          "color": "#ffffff",
        },
        undefined,
      ]
    }
  >
    Not Available
  </Text>
</View>
`;
